phpBB/styles/all/js/webpush.js.twig

```diff
@@ -0,0 +1,287 @@
+/* global phpbb */
```

Comment:
is it really the best way to make the js twig? couldn't we just inject links?

Reply:
When writing it like this I kind of liked the simplicity of not needing this in two separate places. One downside of this current approach is however that linting won't properly work and I guess this will load slower than a plain JS file plus the injection of links.
For the push worker I do however think that this twig approach is the best way of doing it.